Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ScreenData.purs

```diff
@@ -121,7 +121,9 @@ initData = {
         issueFacedView : false,
         issueReason : Nothing,
         issueDescription : "",
-        rideBookingRes : dummyRideBooking
+        rideBookingRes : dummyRideBooking,
+        wasOfferedAssistance : Nothing,
+        donButtonActivate : false 
```

Comment:
Can you recheck this variable name ?

Reply:
Removed as they are not needed, added them for done button visibility but then handled it differently.